### PR TITLE
Unify naming

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

Fixes: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/379

## Type of change

*Please select one of the options below.*

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Discussion

There are still some occurrences of `FDB_INSTANCE_ID` and the locality field (`FDBLocalityInstanceIDKey`)which will be refactored in another PR. 

# Testing

Unit tests + local test

# Documentation

Changed to match the refactoring.

# Follow-up

 https://github.com/FoundationDB/fdb-kubernetes-operator/issues/927

### CODE REVIEW

File: kubectl-fdb/cmd/remove.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * remove.go
```
Comment (brownleej) [inline kubectl-fdb/cmd/remove.go:2]: Since we're not renaming the file, I don't think we want to change this in the header.

**Review by brownleej — CHANGES_REQUESTED** [review]
I have a couple of small suggestions. Otherwise LGTM

File: kubectl-fdb/cmd/remove.go (hunk 1 above)
Comment (johscheuer) [reply kubectl-fdb/cmd/remove.go:2]: Thanks, that was the refactoring. Let me fix that.

**Review by brownleej — APPROVED** [review]
👍

**Review by ltsampros — APPROVED** [review]

**Pull request merged**